frontend/src/components/NewShoot/GNewShootSelectInfrastructure.vue

```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 <template>
-  <v-row class="my-0">
+  <div class="d-flex">
```

Comment:
```suggestion
  <div class="d-flex justify-start align-center">
```
Does not change the behavior but makes it more clear